doc/modules/feature_selection.rst

```diff
@@ -267,9 +283,6 @@ of features non zero.
      Journal of the Royal Statistical Society, 72 (2010)
      http://arxiv.org/pdf/0809.2932
 
-   * F. Bach, "Model-Consistent Sparse Estimation through the Bootstrap"
-     http://hal.inria.fr/hal-00354771/
```

Comment:
yes it's different but when we wrote this code with Gael it was after
studying these papers. It's giving credit... I am +0.5 on keeping the ref.